frontendExternalSystem/src/main/java/com/virtuslab/gitmachete/frontend/externalsystem/settings/MacheteExecutionSettings.java

```diff
@@ -8,7 +8,7 @@
 
 @Data
 @EqualsAndHashCode(callSuper = true)
-@ToString(callSuper = true)
+@ToString(callSuper = false)
```

Comment:
Unrel: `false` is the default, but let's still require it to be explicit in case a class extends sth different then Object.